src/views/v2/rsvp/ari/sidebar/quantity.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * This template renders the RSVP ticket form quantity input.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/sidebar/quantity.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+?>
+<div class="tribe-tickets__rsvp-ar-quantity">
+	<span class="tribe-common-h7">
+		<?php esc_html_e( 'Total Guests', 'event-tickets' ); ?>
```

Comment:
I get a strong feeling this might be something people would want to customize further ("Guests"), can we add a filter for it above?

Reply:
Ideally I feel that if they want to customize, they can override the template right? I'm not super in favor of adding more filters to the templates. In any case we can use some template tag for `Guests` as we do for RSVP when we want to get the label? (something like `tribe_get_rsvp_label_singular()`)